src/app.js

```diff
@@ -33,12 +36,6 @@ const addToDo = (toDo, id, done, trash) => {
   list.insertAdjacentHTML(position, item);
 };
 
-const completeToDo = (element) => {
-  const COMPLETE = 'checkbox';
-  element.classList.toggle(COMPLETE);
-
-  LIST[element.id].done = element.checked;
-};
 /* eslint-disable */
```

Comment:
Hi @Anny85-code, great job you have done on this project. You just have small changes to make to make it perfect 👍🏻  

- [ ] Please avoid using eslint-disable to get around ESlint errors. ESlint errors are usually denoting an underlying problem with the way your code is setup up. It would be nice if you removed this ESLint error and addressed any issues that you may have.

Reply:
Thank you Cathella for your nice review, please if I may, can you guide me on how I can fix the errors I am preventing by disabling the linters?

Reply:
Thank you for removing those eslint disables and fixing the underlying issues.